doc/bipart.xml

```diff
@@ -135,6 +135,48 @@ gap> NrRightBlocks(x);
   </ManSection>
 <#/GAPDoc>
 
+<#GAPDoc Label="DomainOfBipartition">
+  <ManSection>
+    <Attr Name = "DomainOfBipartition" Arg = "x"/>
+    <Returns>A list of positive integers.</Returns>
+    <Description>
+      If <A>x</A> is a bipartition, then <C>DomainOfBipartition</C> returns the
+      <E>domain</E> of <A>x</A>. The domain of <A>x</A> consists of those
```

Comment:
The emphasis should be on the second occurrence of domain, not the first, since it is being defined in the second sentence. Same for codomain below.